plugins/gtav/gtav.cpp

```diff
@@ -41,11 +41,6 @@ void mumble_registerAPIFunctions(void *) {
 void mumble_releaseResource(const void *) {
 }
 
-mumble_version_t mumble_getVersion() {
```

Comment:
Why remove the version completely? I know that this rewrite will probably be compatible with most future versions, but there may come a point where even the new pattern detection mechanism fails and we have to update the plugin again.

If that ever happens, it would be good if this plugin has a version. Otherwise if a user has installed 2 versions of this plugin, Mumble has no way of knowing which one is newer and the working version, correct?

Reply:
Right.

Reply:
On second thought, and I am sorry I didn't realize this on my previous comment, `2.0.0` would be better than `1.0.0` because otherwise this new version wouldn't even be newer than the previous version of this plugin (`1.59.x`).

Reply:
Ah, yeah, that's what I thought.